doc/plots.rst

```diff
@@ -0,0 +1,115 @@
+#####
+Plots
+#####
+
+The **Plots** pane shows the figures and images generated in the :doc:`editor`, :doc:`console` or the :doc:`variableexplorer` during your session, allowing you to interact with them in several ways.
+
+.. image:: images/plots/plots-execution.gif
+   :align: center
+   :alt: Spyder main window showing execution for the Plots pane
+
+|
+
+The figures shown in the Plots pane are those associated with the currently active :guilable:`Console` tab; if you switch consoles, the list of plots displayed (or none at all, if a new console) will change accordingly.
+
+.. image:: images/plots/plots-switch.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing switching consoles
+
+|
+
+
+
+============
+Options Menu
+============
+
+The options menu in the top right of the :guilabel:`Plots` pane offers several ways to customize how your plots are displayed.
+
+
+~~~~~~~~~~~~~~~~~~~~
+Mute inline plotting
+~~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Mute inline plotting` option is enabled by default, preventing your plots from appearing in the Console.
+If you deactivate this option, figures will display in both the Plots pane and the Console.
+
+.. image:: images/plots/plots-inline-plotting.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing plots in the console
+
+|
+
+
+~~~~~~~~~~~~~~~~~
+Show plot outline
+~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Show plot outline` option, off by default, shows a thin stroke surrounding the area of the figure area, which will also appear in the exported images.
+
+.. image:: images/plots/plots-outline.gif
+   :align: center
+   :alt: Spyder's plots pane showing a plot's outline
+
+|
+
+
+~~~~~~~~~~~~~~~~~~~
+Fit plots to window
+~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Fit plots to Window` option, also enabled by default, sizes the figures to match the pane. 
+Disabling it will display plots at their native size, and allow you use the zoom buttons at the top of the pane to scale them manually.
+
+.. image:: images/plots/plots-zoom.gif
+   :align: center
+   :alt: Spyder main window showing plots zoom
+
+|
+
+
+
+===============
+Toolbar Options
+===============
+
+The toolbar at the top of the Plots pane provides several useful features that allow you to interact with your figures.
+For example, you can cycle sequentially through the plot list with the forward and back arrows.
+
+.. image:: images/plots/plots-arrows.gif
+   :align: center
+   :alt: Spyder main window showing plot navigation using arrows.
+
+|
+
+You can also save one or all the plots in the pane as PNGs by clicking the respective icons in the toolbar.
```

Comment:
I think they can also be saved as SVGs. Please confirm if that's the @juanis2112.

Reply:
Okay, but is this really worth going to all the effort to show and explain, especially since this is otherwise already done?